### ISSUE TITLE
Fork an example of todomvc that doesn't use Scala.rx

### ISSUE THREAD

**Issue by stewSquared** [lihaoyi/workbench-example-app#39]

**Comment by lihaoyi** [comment]
yeah we should probably not use scala.rx in the workshop; it's cool but still kind of experimental/obscure. Better to stick to "basic" DOM operations, or introduce people to react if necessary
